lib/ads/interstitial_ad_manager.js

```diff
@@ -675,11 +675,25 @@ shaka.ads.InterstitialAdManager = class {
     });
     try {
       this.updatePlayerConfig_();
+      // playRangeEnd in src= causes the ended event not to be fired when that
```

Comment:
I don't understand this comment in context of the next paragraph of code.  Could you improve the comments in this whole try/catch?  Happy to have a conversation about it if it helps you figure out what is missing from the comments.

Reply:
For src= we use https://www.w3.org/TR/media-frags/ whose implementation in Safari implies that when the playRangeEnd time is reached, if the stream has not finished, what it does is pause the content, but does not emit the ended event.

Reply:
IIUC, you are saying that this trick of using play range to limit the interstitial will not trigger an inappropriate 'ended' event, so we can then go back to the main content.  Is that correct?

If that's right, I would add to this comment:

"So we can still return to the main content after that."